provisioning_templates/provision/kickstart_default.erb

```diff
@@ -149,10 +149,10 @@ skipx
 
   # VLAN (only on physical is recognized)
   if iface.virtual? && iface.tag.present? && iface.attached_to.present?
-    if rhel_compatible && os_major >= 6
-      network_options.push("--vlanid=#{iface.tag}")
-    elsif rhel_compatible && os_major >= 7
+    if rhel_compatible && os_major >= 7
       network_options.push("--interfacename=vlan#{iface.tag}")
+    elsif rhel_compatible && os_major >= 6
```

Comment:
shouldn't this be 
```suggestion
    elsif rhel_compatible && os_major == 6
```
since >= 7 is already handled earlier?
Or even simpler, change the first condition to
```
if rhel_compatible && os_major == 6
```
and then you don't need to modify anything else

Reply:
Yeah it should be `<=`. Honestly I have only checked RHEL6 docs, not sure if RHEL5 supports VLAN. Probably?

Reply:
Let's keep it as == then to preserve the original behaviour, just in case someone still provisions EL5 for some odd reason.

Reply:
Okay.